tools_for_todoist/models/event.py

```diff
@@ -100,6 +100,14 @@ def get_private_info(self, key):
             return None
         return self._extended_properties.get('private', {}).get(key)
 
+    def delete_private_info(self, key):
```

Comment:
I don't think this is necessary, we always keep `last_completed` set to some value (even during initial processing)